### PR TITLE
Final changes to the assessment. 

### PR DESCRIPTION
**Pull request by 1point618**

### CODE REVIEW

**Comment by 1point618** [comment]
@crichID You may want to look at this PR as well as the other two I linked you to.

File: modules/DSA-02_project-1.yml [hunk 1]
```diff
@@ -26,13 +26,13 @@ screens:
               id: DSA-02-new-content-verification
               success-message: "Great job - you successfully updated the file."
               failure-message: "It looks like you haven't updated the file yet."
-        - description: "Make sure the change shows up under Pull Request #5. If it does, make a comment in the PR about the update."
+        - description: "Create a Pull Request to merge the `add-gitignore` branch into `master`."
           id: DSA-02-pr-comment
```
Comment (crichID) [inline modules/DSA-02_project-1.yml:30]: @1point618 dumb question - does this id need to change based on the different verification required here (a comment vs. a new PR)? I would guess the id is irrelevant to the actual verification, but wanted to double check.

**Comment by crichID** [comment]
@1point618 one very small question, but otherwise this one looks good!

File: modules/DSA-02_project-1.yml (hunk 1 above)
Comment (1point618) [inline modules/DSA-02_project-1.yml:30]: Right, the ID is completely irrelevant in terms of what the validation does. It does have some implications for how this is imported over old data, and so it's easier for now to keep the same ID.

**Pull request merged**